lib/openstack_query/docs/user_docs/API.md

```diff
@@ -48,6 +48,9 @@ from enums.query.props.server_properties import ServerProperties
 # create a ServerQuery and find server names and ids
 query = ServerQuery()
 query.select(ServerProperties.SERVER_NAME, ServerProperties.SERVER_ID)
+
+# or using the equivalent string aliases
+query.select("server_name", "server_id")
```

Comment:
Too late now, but IMO this should be a list since every other API isn't implicitly converted to a list and it's not very Pythonic since I read it as two parameters

Reply:
it is two parameters though? I don't see what's wrong here

Reply:
I'll do this in a separate PR